adapters/appnexus/params_test.go

```diff
@@ -50,6 +50,7 @@ var validParams = []string{
 	`{"placementId":123, "keywords":[{"key":"foo","value":["bar"]}]}`,
 	`{"placement_id":123, "keywords":[{"key":"foo","value":["bar", "baz"]}]}`,
 	`{"placement_id":123, "keywords":[{"key":"foo"}]}`,
+	`{"placement_id":123, "keywords":{"genre": ["rock", "pop"], "pets": ["dog"]}}`,
```

Comment:
Could you please also add a test case where keywords is just a string?

Reply:
``{"placement_id":123, "keywords":"foo=bar,foo=baz"}`,` This case covers the test case where Keywords is a string